File: weather-api/src/main/java/com/thewizrd/weather_api/aqicn/AttributionsItem.java
package com.thewizrd.weather_api.aqicn;

import com.google.gson.annotations.SerializedName;

public class AttributionsItem {

    @SerializedName("name")
    private String name;

    @SerializedName("url")
    private String url;

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getUrl() {
        return url;
    }
}

File: weather-api/src/main/java/com/thewizrd/weather_api/aqicn/AttributionsItem.kt
package com.thewizrd.weather_api.aqicn

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AttributionsItem(
    @Json(name = "name")
    var name: String? = null,

    @Json(name = "url")
    var url: String? = null
)